documentation/Logging.md

```diff
@@ -0,0 +1,15 @@
+# Docker
+
+We have logging set up. You can change the log level by setting the environment variable `LOG_LEVEL` to one of:
+- DEBUG
+- INFO
+- WARNING
+- CRITICAL
```

Comment:
I believe there's a tier between these of ERROR which is also fairly common - it's pretty rare we'd want to discount those and _just_ show critical ones